data/brands/shop/pastry.json

```diff
@@ -122,6 +122,18 @@
         "shop": "pastry"
       }
     },
+    {
+      "displayName": "Lola's Cupcakes",
+      "locationSet": {
+        "include": ["gb-lon.geojson"]
```

Comment:
Despite the website saying London a lot, they definitely have stores listed in Milton Keynes, Cambridge and Birmingham, so I think this should be gb-eng at the tightest:
https://www.lolascupcakes.co.uk/cupcakes/stores

There are also two international franchises listed in Japan:
https://www.lolascupcakes.co.uk/InternationalStores.aspx